instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientMetrics.java

```diff
@@ -41,16 +47,24 @@ public static OperationMetrics get() {
     return HttpClientMetrics::new;
   }
 
-  private final DoubleHistogram duration;
+  @Nullable private final DoubleHistogram stableDuration;
+  @Nullable private final DoubleHistogram oldDuration;
 
   private HttpClientMetrics(Meter meter) {
-    String durationInstrumentName =
-        HttpMetricsUtil.emitNewSemconvMetrics
-            ? "http.client.request.duration"
-            : "http.client.duration";
-    duration =
-        createDurationHistogram(
-            meter, durationInstrumentName, "The duration of the outbound HTTP request");
+    if (SemconvStability.emitStableHttpSemconv()) {
+      stableDuration =
+          createDurationHistogram(
+              meter, "http.client.request.duration", "The duration of the outbound HTTP request");
```

Comment:
I think you'll also need to change the condition on which the advice is applied: right now the buckets are not set if `emitOldHttpSemconv` is true, and that does not really make sense when we split the histograms.

Reply:
(just making sure: missing commit?)

Reply:
🙈